CHANGELOG.md

```diff
@@ -41,6 +41,8 @@ CHANGELOG
 ### Packaging
 
 ### Tools
+- `intelmqsetup`:
+  - SECURITY: fixed a low-risk bug causing the tool to change owner of `/` path if running with the `INTELMQ_PATHS_NO_OPT` environment variable set to `True`. This affects only the PIP package as the DEB package doesn't contain this tool. (PR#2355 by Kamil Mańkowski)
```

Comment:
The value of `INTELMQ_PATHS_NO_OPT` does not matter as long as it is set (the code checks for existence).
```suggestion
  - SECURITY: fixed a low-risk bug causing the tool to change owner of `/` if run with the `INTELMQ_PATHS_NO_OPT` environment variable set. This affects only the PIP package as the DEB/RPM packages don't contain this tool. (PR#2355 by Kamil Mańkowski, fixes #2354)
```